requirements.txt

```diff
@@ -2,3 +2,4 @@ jinja2
 mathlibtools
 networkx
 gitpython
+requests
```

Comment:
This is no longer used, right? Instead, we need the GitHub API here

Reply:
remove this, also from requirements and also `logging` import